3_visualize/src/visualize.R

```diff
@@ -0,0 +1,60 @@
+#' Create a plot
+#' 
+#' @export figure_1 png, viz depicts training temp profile (#) by test RMSE (°C) for three model types 
+
+
+visualize <- function(){
+  
+  # Check for existence of out subfolder
+  dir.exists("3_visualize/out")
+  
+  # Read in eval data
+  eval_data <- readRDS("2_process/out/eval_data")
+  
+  png(file = file.path('3_visualize/out/figure_1.png'), width = 8, height = 10, res = 200, units = 'in')
```

Comment:
Similar to colors and `pch`, these plot variables (width, height, res, units) are good candidates for arguments